### ISSUE TITLE
About the input format in ConsolePlayer

### ISSUE THREAD

**Issue by Chen-Mengqi** [CastleLab/COMP3021-2020Fall-PA1-Student-Version#17]
In my program, I only take the format like "a1->b2" as valid, no space shall be in the string.
But in PA1_obfuscated.jar, the program take "a1 ->   b2  " also as valid.
So must I also take the format mentioned above valid? Thank you.

**Comment by Troublor** [comment]
Yes, your `ConsolePlayer` should handle as many variants of inputs as possible.
